packages/nuxt/src/core/runtime/nitro/paths.ts

```diff
@@ -1,4 +1,4 @@
-import { joinURL } from 'ufo'
+import { joinRelativeURL as joinURL } from 'ufo'
```

Comment:
I would suggest using `joinRelativeURL` as the canonical name to make today's (and future) tracing easier.